.prettierrc

```
@@ -0,0 +1,5 @@
+{
+  "printWidth": 120,
```

Nitpick, but I’d suggest using the default 80 here.

```suggestion
  "printWidth": 80,
```

---

Done.